CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# CorisxTH Contributing Guidelines #
+
+## Tell other devs what you are working on! ##
+
+When you start working on a fix/enhancement for an open issue please always
+post a comment in this issue's discussion to tell other devs that you have
+started working on it so that they won't waste any of their free time by
+working on their own fix/enhancement for it.
+
+If there isn't an open issue discussion for a bug fix/enhancement you want
+to work on please check that there isn't a closed discussion for it because
+a previous developer of it didn't finish their work which you could finish.
+
+And if there is no existing issue discussion for what you want to work on
+please open a new issue for it and tell other devs here that your going to/
+have started working on it.
+
+## Enhancement/Suggestions ##
+If you want to make a suggestion, please use the *Feature Request* option when making a new issue. It will provide all the information needed to evaluate the suggestion or enhancement.\
```

Comment:
```suggestion

If you want to make a suggestion, please use the *Feature Request* option when making a new issue. It will provide all the information needed to evaluate the suggestion or enhancement.\
```